### DISCUSSION TITLE
Problemas con la Aplicacion IOS

### DISCUSSION THREAD

**Discussion by tonyvilla90** [firefly-iii/firefly-iii#7614] [Firefly III]
hola buenas quisiera reportar un problema el cual sucede con la aplicacion de iphones en mi caso IOS, al conectar con el sistema la aplicacion se pone blanca la pantalla completamemte y no fumciona, la unica solucione s desintalar la aplicacion y volver a instalarla y no poder vincularla nuevemente ya q cada vez que lo hago pasa lo mismo 

Saludos cordiales

**Comment by JC5** [comment]
English please.
